pkgs/development/python-modules/jxlpy/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  cython,
+  libjxl,
+}:
+buildPythonPackage rec {
+  pname = "jxlpy";
+  version = "0.9.5";
+
+  format = "pyproject";
```

```suggestion
  pyproject = ture;
```
`format` is deprecated. Could you set `pyproject` instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters